migrations/13_refactor_tasks.sql

```diff
@@ -0,0 +1,39 @@
+ALTER TABLE tasks ALTER COLUMN tsk_status TYPE TEXT;
+DROP TYPE task_status;
+
+CREATE TYPE task_status AS ENUM (
+    'success',
+    'pending',
+    'failed'
+);
+
+CREATE TYPE mutability AS ENUM (
+    'immutable',
+    'mutable'
+);
+
+ALTER TABLE tasks RENAME COLUMN tsk_id TO metadata_hash;
+ALTER TABLE tasks RENAME COLUMN tsk_metadata_url TO metadata_url;
+
+ALTER TABLE tasks
```

Comment:
In general prefixing a column name is a good practice, that we've already broken at least once. Why it's a good practice imo:
- clarity and verbosity in a query - where is this column from becomes obvious
- no naming collisions